src/app/copy/inline-copy/inline-copy.component.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewEncapsulation
+} from '@angular/core';
+
+import { CopyService } from '../copy-service/copy.service';
+
+@Component({
+  encapsulation: ViewEncapsulation.None,
+  selector: 'pfng-inline-copy',
+  templateUrl: './inline-copy.component.html',
+  styleUrls: ['./inline-copy.component.less']
+})
+
+export class InlineCopyComponent implements OnInit {
+  @Input('label') label: string;
```

Comment:
Sure I see, I was using label here because it aligned nicely with the same `label` input property for the block level copy component but it's a good point about it not actually being label text, I'll update this.

Allowing for custom tooltip text sounds like a good idea, I'll just set it to the ariaLabel value by default, and users can override if they like.